recipes/ncvue/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "ncvue" %}
+{% set version = "4.1.1" %}
+{% set sha256 = "fc1d5769e80462c8d2edd166f73eaea28187f463114c7335dab46e33f5d54ce8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ncvue-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  script_env:
+   - MPLBACKEND=TkAgg
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools >=42
+    - setuptools_scm >=3.5
+  run:
+    - cartopy
+    - cython
+    - numpy
+    - matplotlib-base
+    - netcdf4
+    - pykdtree
+    - pyshp
+    - python
+    - scipy
+    - shapely
+    - six
+    - tk  # [linux]
+
+test:
+  imports:
+    - ncvue
+  commands:
+    - pip check
+  requires:
+    - pip
+    - xorg-libx11  # [linux]
```

Comment:
```suggestion
```
You need to include a `yum_requirements.txt` file if you want to load any libraries that link against opengl.

https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl

Reply:
Hey @carterbox

I added a yum_requirements.txt file with the entries:
```
mesa-libGL
mesa-dri-drivers
libselinux
libXdamage
libXxf86vm
libXext
xorg-x11-server-Xorg
```
But the build still fails with the messsage:
`ImportError: Cannot load backend 'TkAgg' which requires the 'tk' interactive framework, as 'headless' is currently running`
???

Reply:
Sorry, but I'm not familiar with the various backends for matplotlib or their requirements. What happened when you switched to Agg the logs are missing?

Are you trying to open GUI elements on our headless servers?